### ISSUE TITLE
PlayerDB Stand not reliably removed at high elevations

### ISSUE THREAD

**Issue by GrifterMage** [rx-modules/PlayerDB#13]
`rx.playerdb/functions/impl/get_name.mcfunction` summons a temporary armor stand to get the name of the player invoking it in plain text, then gets rid of it. However, it does so by teleporting the stand to a relative position that is not reliably outside the world; calling the function at high Y values can result in the stand not being teleported far enough below Y0 to ensure deletion.

Relevant section:
```
#> get rid of stand w/o killing it so the UUID is available

tp e3241b58-07c0-5230-9794-aecdc5aaaad4 ~ ~-100 ~
```

**Comment by RitikShah** [comment]
I think I've resolved this by changing `~-100` to `~-2500`

**Comment by RitikShah** [comment]
(Fixed in: 4e4ec42ce38ded5eecade60aa9861327e695b372)
